include/hse/hse.h

```diff
@@ -674,10 +674,9 @@ hse_kvs_param_get(
  * @param flags: Flags for operation specialization.
  * @param txn: Transaction context (optional).
  * @param pfx: Prefix of keys to delete.
- * @param pfx_len: Length of @p pfx.
+ * @param pfx_len: Length of @p pfx, must equal KVS prefix length.
```

Comment:
Make the second part a remark

`@remark @p pfx_len must equal the prefix length of @p kvs`

Reply:
You didn't respond to this :(

Reply:
didn't even see it :(